locales/en-US/app.ftl

```diff
@@ -568,6 +626,66 @@ TrackContextMenu--hide-other-screenshots-tracks = Hide other Screenshots tracks
 TrackContextMenu--hide-track = Hide “{ $trackName }”
 TrackContextMenu--show-all-tracks = Show all tracks
 
+## TransformNavigator
+## Navigator for the applied transforms in the Call Tree, Flame Graph, and Stack
+## Chart components.
+## To learn more about them, visit:
+## https://profiler.firefox.com/docs/#/./guide-filtering-call-trees?id=transforms
+
+# Root item in the transform navigator.
+# See: https://profiler.firefox.com/docs/#/./guide-filtering-call-trees?id=collapse
+# Variables:
+#   $item (String) - Name of the current thread. E.g.: Web Content.
+TransformNavigator--complete = Complete “{ $item }”
+
+# "Collapse resource" transform.
+# See: https://profiler.firefox.com/docs/#/./guide-filtering-call-trees?id=collapse
+# Variables:
+#   $item (String) - Name of the resource that collapsed. E.g.: libxul.so.
+TransformNavigator--collapse-resource = Collapse: { $item }
```

Comment:
Could you show me where all the following strings appear? I have a hard time understanding if they are verb or nouns.

Reply:
Yeah they are a bit hard to discover. They appear on the right side of `Complete "xyz"` when you add more transforms. 
STR:
1. Open [this profile](https://deploy-preview-3393--perf-html.netlify.app/public/hrp3j3vckxd1193cwzea0j7hpbne4n3wp1fc75r/calltree/?globalTrackOrder=7-0-1-2-3-4-5-6&hiddenGlobalTracks=2-3&hiddenLocalTracksByPid=9832-1-2-4~6032-2&localTrackOrderByPid=9832-6-7-0-1-2-3-4-5~9956-1-0~2516-1-0~3672-1-0~5372-1-0~3636-1-2-0~6032-4-0-1-2-3~&profileName=Firefox%2088%20%E2%80%93%20Windows%2010%20-%20Fast%20Reference%20Laptop%20-%20Warm%20Startup&thread=0&timelineType=cpu-category&v=5).
2. In the Call Tree panel, right click on a node and click on "Merge Function" in the context menu that appears.
3. You'll see an item is appended to the right side of the  `Complete "Parent Process"`. These new strings are these items on this navigation bar.
4. Right click and select the other options to see other transforms.

You can also click on any of these navigation items to go back now. You can combine different transforms on top of each other and they will, well, :transform" the Call Tree with these different type of actions.